cmd/buildkitd/config/config.go

```diff
@@ -154,10 +154,24 @@ type ContainerdRuntime struct {
 }
 
 type GCPolicy struct {
-	All          bool      `toml:"all"`
-	KeepBytes    DiskSpace `toml:"keepBytes"`
-	KeepDuration Duration  `toml:"keepDuration"`
-	Filters      []string  `toml:"filters"`
+	All     bool     `toml:"all"`
+	Filters []string `toml:"filters"`
+
+	KeepDuration Duration `toml:"keepDuration"`
+
+	// KeepBytes is a synonym for MaxStorage
+	KeepBytes DiskSpace `toml:"keepBytes"`
```

Comment:
We should mark this deprecated if it is replaced.

Reply:
Done.